### ISSUE TITLE
Built-in datasets: Convert still fails

### ISSUE THREAD

**Issue by BrianWestlake** [mila-udem/fuel#377]
I get this result when I try to convert the celeba dataset

```
fuel-convert celeba 64

Traceback (most recent call last):
  File "/usr/local/bin/fuel-convert", line 11, in <module>
    load_entry_point('fuel==0.2.0', 'console_scripts', 'fuel-convert')()
  File "/usr/local/lib/python2.7/dist-packages/fuel/bin/fuel_convert.py", line 79, in main
    output_paths = convert_function(**args_dict)
  File "/usr/local/lib/python2.7/dist-packages/fuel/converters/celeba.py", line 198, in convert_celeba
    directory, output_directory, output_filename)
  File "/usr/local/lib/python2.7/dist-packages/fuel/converters/base.py", line 45, in wrapped
    return f(directory, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/fuel/converters/celeba.py", line 139, in convert_celeba_64
    h5file = _initialize_conversion(directory, output_path, (64, 64))
  File "/usr/local/lib/python2.7/dist-packages/fuel/converters/celeba.py", line 41, in _initialize_conversion
    skiprows=2, usecols=tuple(range(1, 41))) +
  File "/usr/local/lib/python2.7/dist-packages/numpy/lib/npyio.py", line 923, in loadtxt
    vals = [vals[i] for i in usecols]
IndexError: list index out of range
```

This is with the latest versions of Theano, Blocks and Fuel

**Comment by Wakeupbuddy** [comment]
I got the same problem

**Comment by rizar** [comment]
@vdumoulin , I confirm that `fuel-convert celeba` is broken. Would you like to take a look?

**Comment by vdumoulin** [comment]
I'll take a look, thanks for pointing it out.

**Comment by vdumoulin** [comment]
`fuel-convert celeba 64` works fine, the problem is that the links for the data files have changed and need to be updated in order for `fuel-download celeba` to work properly.

I opened a PR with a fix (#378). @rizar, would you mind giving it a quick review?
